src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -723,6 +729,22 @@ public static Transaction createAssetTransferTransaction(// AssetTransaction
         return tx;
     }
 
+    /** Lease enforces mutual exclusion of transactions.  If this field
+     * is nonzero, then once the transaction is confirmed, it acquires
+     * the lease identified by the (Sender, Lease) pair of the
+     * transaction until the LastValid round passes.  While this
+     * transaction possesses the lease, no other transaction
+     * specifying this lease can be confirmed. 
+     * The Size is fixed at 32 bytes. 
+     * @param lease 32 byte lease
+     **/
+    public void setLease(byte[] lease) {
+        if (lease.length != 32 && lease.length != 0) {
```

Comment:
Consider making it a constant to avoid magic numbers

Reply:
Fixed!